README.md

```diff
@@ -1,10 +1,10 @@
 c-utf8
 ======
 
-UTF-8 Handling in Standard ISO-C11
+UTF-8 Handling in standard ISO C11
```

Comment:
Yeah, we sometimes use `title case` here, sometimes we don't. Either way, I think `handling` and `standard` should follow the same capitalization rules, shouldn't they?